custom/icds/templates/icds/messaging/indicators/kan/aww_vhnd_survey.txt

```diff
@@ -1 +1 @@
-ದಾಖಲೆಯ ಪ್ರಕಾರ, ಹಿಂದಿನ ತಿಂಗಳಲ್ಲಿ ನಿಮ್ಮ ಕೇಂದ್ರದಲ್ಲಿ ವಿಎಚ್‌ಎಸ್‌ಎನ್‌ಡಿ ಸಂಭವಿಸಿಲ್ಲ. ಇದನ್ನು ಪ್ರತಿ ತಿಂಗಳು ನಿಯಮಿತವಾಗಿ ನಡೆಸಲಾಗಿದೆಯೆ ಎಂದು ಖಚಿತಪಡಿಸಿಕೊಳ್ಳಿ.
\ No newline at end of file
+ ದಾಖಲೆಯ ಪ್ರಕಾರ, ಹಿಂದಿನ ತಿಂಗಳಲ್ಲಿ ನಿಮ್ಮ ಕೇಂದ್ರದಲ್ಲಿ ವಿಎಚ್‌ಎಸ್‌ಎನ್‌ಡಿ ಆಗಲು ಸಾಧ್ಯವಿಲ್ಲ. ಇದನ್ನು ಪ್ರತಿ ತಿಂಗಳು ನಿಯಮಿತವಾಗಿ ನಡೆಸಲಾಗಿದೆಯೆ ಎಂದು ಖಚಿತಪಡಿಸಿಕೊಳ್ಳಿ.
```

Comment:
Hey @akshita-sh,
Great job in sending in the PR. 👍 
I can see one space at the starting of the line, I just wanted to know is this intentional?

Reply:
great catch @AmitPhulera

Reply:
Nope. Seems like an oversight from me. Should I create a new PR for this or can this be edited on this one?

Reply:
No need to create a new PR, you can make edits in this PR by adding a new commit.
Feel free to reach out if you need help with it.

Reply:
Thanks @AmitPhulera. Removed the blank